entities/bug/timeline.go

```diff
@@ -74,6 +78,10 @@ func (c *CommentTimelineItem) Edited() bool {
 	return len(c.History) > 1
 }
 
+func (c *CommentTimelineItem) DeletedBy() identity.Interface {
+	return c.History[len(c.History)-1].Author
```

Comment:
That doesn't seem robust enough. What if something else happen after deleting? At the moment, it seems like only the local termui would prevent that. Going further, someone running another version of git-bug might make more change on that deleted comment.

We also need to decide what is the semantic to enforce here. If a comment is deleted, do we make no-op any further change? Do we accept them and gracefully handle that (as in, AddComment+DeleteComment+EditComment+UndeleteComment would show the modified comment). I'd go for the latter.